main.py

```diff
@@ -32,13 +32,17 @@ def generate_features(stances,dataset,name):
 
 if __name__ == "__main__":
     check_version()
-    parse_params()
+    is_full_set = parse_params()
```

Comment:
How do you know `is_full_set` points to either default or `-r`? What is this value if users use `-c`?

Reply:
if the user use `-c`, it will return `True`
if the user use `-r`, it will return `False`
if the user use '-c -r`, it will return `False`
if the user use nothing , it will return `True` 
so i think there is no problem here

Reply:
I see from the other piece. 👍